Externals/luajit.cmake

```
@@ -6,340 +6,347 @@
+set(LUAI_MAXSTACK 65500 CACHE STRING "Max. # of stack slots for a thread (<64K).")
+set(LUAI_MAXCSTACK 8000 CACHE STRING "Max. # of stack slots for a C func (<10K).")
+set(LUAI_GCPAUSE 200 CACHE STRING "Pause GC until memory is at 200%.")
+set(LUAI_GCMUL 200 CACHE STRING "Run GC at 200% of allocation speed.")
+set(LUA_MAXCAPTURES 32 CACHE STRING "Max. pattern captures.")
 
 ## SETUP
-set ( LUA_DIRSEP "/" )
-set ( LUA_MODULE_SUFFIX ${CMAKE_SHARED_MODULE_SUFFIX} )
-set ( LUA_LDIR ${INSTALL_LMOD} )
-set ( LUA_CDIR ${INSTALL_CMOD} )
-
-if ( LUA_USE_RELATIVE_LOADLIB )
-  # This will set up relative paths to lib
-  string ( REGEX REPLACE "[^!/]+" ".." LUA_DIR "!/${INSTALL_BIN}/" )
-else ()
-  # Direct path to installation
-  set ( LUA_DIR ${CMAKE_INSTALL_PREFIX} CACHE STRING "Destination from which modules will be resolved. See INSTALL_LMOD and INSTALL_CMOD.")
-endif ()
-
-set ( LUA_PATH_DEFAULT "./?.lua;${LUA_DIR}${LUA_LDIR}/?.lua;${LUA_DIR}${LUA_LDIR}/?/init.lua;./?/init.lua" )
-set ( LUA_CPATH_DEFAULT "./?${LUA_MODULE_SUFFIX};${LUA_DIR}${LUA_CDIR}/?${LUA_MODULE_SUFFIX};${LUA_DIR}${LUA_CDIR}/loadall${LUA_MODULE_SUFFIX}" )
-
-if ( WIN32 AND NOT CYGWIN )
-  # Windows systems
-  add_definitions ( -DLUA_BUILD_AS_DLL -DLUAJIT_OS=LUAJIT_OS_WINDOWS)
-  option ( LUA_BUILD_WLUA "Build wluajit interpreter for no-console applications." ON )
-  set ( LJVM_MODE peobj )
-  # Paths (Double escapes needed)
-  set ( LUA_DIRSEP "\\\\" )
-  string ( REPLACE "/" ${LUA_DIRSEP} LUA_DIR "${LUA_DIR}" )
-  string ( REPLACE "/" ${LUA_DIRSEP} LUA_LDIR "${LUA_LDIR}" )
-  string ( REPLACE "/" ${LUA_DIRSEP} LUA_CDIR "${LUA_CDIR}" )
-  string ( REPLACE "/" ${LUA_DIRSEP} LUA_PATH_DEFAULT "${LUA_PATH_DEFAULT}" )
-  string ( REPLACE "/" ${LUA_DIRSEP} LUA_CPATH_DEFAULT "${LUA_CPATH_DEFAULT}" )
-
-elseif ( APPLE )
-  set ( CMAKE_EXE_LINKER_FLAGS "-pagezero_size 10000 -image_base 100000000 -image_base 7fff04c4a000" )
-  option ( LUA_USE_POSIX "Use POSIX functionality." ON )
-  option ( LUA_USE_DLOPEN "Use dynamic linker to load modules." ON )
-  set ( LJVM_MODE machasm )
-else ()
-  option ( LUA_USE_POSIX "Use POSIX functionality." ON )
-  option ( LUA_USE_DLOPEN "Use dynamic linker to load modules." ON )
-  set ( LJVM_MODE elfasm )
-endif ()
+set(LUA_DIRSEP "/")
+set(LUA_MODULE_SUFFIX ${CMAKE_SHARED_MODULE_SUFFIX})
+set(LUA_LDIR ${INSTALL_LMOD})
+set(LUA_CDIR ${INSTALL_CMOD})
+
+if(LUA_USE_RELATIVE_LOADLIB)
+    # This will set up relative paths to lib
+    string(REGEX REPLACE "[^!/]+" ".." LUA_DIR "!/${INSTALL_BIN}/")
+else()
+    # Direct path to installation
+    set(LUA_DIR ${CMAKE_INSTALL_PREFIX} CACHE STRING "Destination from which modules will be resolved. See INSTALL_LMOD and INSTALL_CMOD.")
+endif()
+
+set(LUA_PATH_DEFAULT "./?.lua;${LUA_DIR}${LUA_LDIR}/?.lua;${LUA_DIR}${LUA_LDIR}/?/init.lua;./?/init.lua")
+set(LUA_CPATH_DEFAULT "./?${LUA_MODULE_SUFFIX};${LUA_DIR}${LUA_CDIR}/?${LUA_MODULE_SUFFIX};${LUA_DIR}${LUA_CDIR}/loadall${LUA_MODULE_SUFFIX}")
+
+if(WIN32 AND NOT CYGWIN)
```

There should be a space after all `if`'s and `elseif`'s
```suggestion
if (WIN32 AND NOT CYGWIN)
```